osmosis-apidb/src/main/java/org/openstreetmap/osmosis/apidb/v0_6/impl/ChangesetManager.java

```diff
@@ -143,8 +143,7 @@ private void addChangeset(long changesetId, long userId) {
 
             // Insert the changeset tags.
             prmIndex = 1;
-            insertTagStatement.setLong(prmIndex++, changesetId);
-            insertTagStatement.setLong(prmIndex++, changesetId);
+            insertTagStatement.setLong(++prmIndex, changesetId);
```

Comment:
@rory Here I removed extra line which was doing the same and changed to ++prmIndex which first increments and then passes argument opposite to what was before.